examples/tests/test_03_message_provider.py

```diff
@@ -0,0 +1,59 @@
+"""
+Test Message Pact provider.
+
+Unlike the standard Pact, which is designed for HTTP interactions, the Message
+Pact is designed for non-HTTP interactions. This example demonstrates how to use
+the Message Pact to test whether a provider generates the correct messages.
```

Comment:
>the Message Pact

MessagePact?

Reply:
`MessagePact` would be the class name, but in regular English I would refer to it as "Message Pact" (same goes for the `pact-python` vs Pact Python). The space is also consistent with the docs you linked above.